app/backend/src/core/modules/pricing/PriceUpdaterModule.ts

```diff
@@ -0,0 +1,40 @@
+import Logger from "@/tools/Logger";
+import { Database } from "@/database/Database";
+import CoinCapClient from "@/core/clients/coincap/CoinCapClient";
+import { PriceUpdaterController } from "../../controllers/pricing/PriceUpdaterController";
+import { PriceRepository } from "../../../database/repositories/PricingRepository";
+import TaskScheduler from "../../../core/scheduler/TaskScheduler";
+import { Config } from "./../../../config/Config";
+
+export function createPriceUpdaterModule(
+  config: Config,
+  logger: Logger,
+  database: Database,
+  coinCapClient: CoinCapClient,
+): () => Promise<void> {
+  const loggerContext: string = "Price updater module";
+
+  const priceRepository = new PriceRepository(database.getKnex());
+
+  const priceUpdaterController = new PriceUpdaterController(
+    coinCapClient,
+    priceRepository,
+    logger,
+  );
+
+  const priceUpdaterTaskScheduler = new TaskScheduler(
+    () => priceUpdaterController.start(),
+    10000,
```

Comment:
Do we really need to run it every 10 seconds given that we use 15m CoinCap intervals?